model/role.go

```diff
@@ -632,6 +737,26 @@ func MakeDefaultRoles() map[string]*Role {
 							PERMISSION_LIST_PRIVATE_TEAMS.Id,
 							PERMISSION_JOIN_PRIVATE_TEAMS.Id,
 							PERMISSION_VIEW_MEMBERS.Id,
+							PERMISSION_READ_SYSCONSOLE_ABOUT.Id,
+							PERMISSION_WRITE_SYSCONSOLE_ABOUT.Id,
+							PERMISSION_READ_SYSCONSOLE_REPORTING.Id,
+							PERMISSION_WRITE_SYSCONSOLE_REPORTING.Id,
+							PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT.Id,
+							PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT.Id,
+							PERMISSION_READ_SYSCONSOLE_ENVIRONMENT.Id,
+							PERMISSION_WRITE_SYSCONSOLE_ENVIRONMENT.Id,
+							PERMISSION_READ_SYSCONSOLE_SITE.Id,
+							PERMISSION_WRITE_SYSCONSOLE_SITE.Id,
+							PERMISSION_READ_SYSCONSOLE_AUTHENTICATION.Id,
+							PERMISSION_WRITE_SYSCONSOLE_AUTHENTICATION.Id,
+							PERMISSION_READ_SYSCONSOLE_PLUGINS.Id,
+							PERMISSION_WRITE_SYSCONSOLE_PLUGINS.Id,
+							PERMISSION_READ_SYSCONSOLE_INTEGRATIONS.Id,
+							PERMISSION_WRITE_SYSCONSOLE_INTEGRATIONS.Id,
+							PERMISSION_READ_SYSCONSOLE_COMPLIANCE.Id,
+							PERMISSION_WRITE_SYSCONSOLE_COMPLIANCE.Id,
+							PERMISSION_READ_SYSCONSOLE_EXPERIMENTAL.Id,
+							PERMISSION_WRITE_SYSCONSOLE_EXPERIMENTAL.Id,
 						},
 						roles[TEAM_USER_ROLE_ID].Permissions...,
 					),
```

Comment:
TODO: Verify each of the non-system-consoler permissions for each of the new roles.